pages.fr/common/shards.md

```diff
@@ -0,0 +1,24 @@
+# shards
+
+> Outil de gestion des dépendances pour le langage Crystal.
+> Plus d'informations : <https://crystal-lang.org/reference/the_shards_command>.
+
+- Créé un fichier `shard.yml` squelette :
+
+`shards init`
+
+- Installe les dépendances à partir d'un fichier `shard.yml` :
+
+`shards install`
+
+- Met à jour toutes les dépendances :
+
+`shards update`
+
+- Liste toutes les dépendances installées :
+
+`shards list`
+
+- Liste la version de la dépendance :
```

Comment:
Suggestions, but unsure because the documentation is not "crystal clear": 😉 

> shards version: Shows version of a shard

and

> Prints the version of the shard.

```suggestion
- Liste la version d'une dépendance :
```
or 
```suggestion
- Liste la version d'une dépendance ayant un chemin donné :
```